homeassistant/components/simplisafe/__init__.py

```diff
@@ -474,6 +475,19 @@ async def async_init(self):
                 _LOGGER.error("Error while fetching initial event: %s", err)
                 self.initial_event_to_use[system.system_id] = {}
 
+            # The initial update doesn't process notifications because it is likely that
+            # they will trigger SIMPLISAFE_NOTIFICATION events before other critical
+            # components – such as automation – are ready. Therefore, we schedule an
+            # update of notifications as soon as HASS is fully started:
+            async def process_notifications(_):
+                while self._hass.state != CoreState.running:
+                    await asyncio.sleep(1)
```

Comment:
Home Assistant will wait for all event listeners to finish before going from starting to running with a timeout of 10 seconds. Like this we'll trigger this timeout and print a warning because this task will never finish.

Reply:
I'm not experiencing this in my tests, but I'll defer to your knowledge How, then, do I solve the problem stated above?

Reply:
Use `hass.loop.create_task()` to not be tracked.

Reply:
In Python 3.7 that should be `asyncio.create_task`.